### PR TITLE
Remove DOCKER_LOG in favor of LOG_TO_STDOUT

### PR DESCRIPTION
**Pull request by kushagharahi**
Context https://github.com/haugene/docker-transmission-openvpn/pull/1379

### CODE REVIEW

**Comment by pkishino** [comment]
Can you please rebase this to remove the other health check commit ?

**Comment by haugene** [comment]
I think this might also be due to master being commits before dev. I'll create a PR that merges master into dev and the result of that is probably what @kushagharahi should rebase on top of. I think we can keep most of the changes from master but since one Dockerfile (for alpine) is removed on dev and we'll deprecate DOCKER_LOG the last fixups has to be done there.

**Comment by kushagharahi** [comment]
ok @haugene and @pkishino I've rebased off of the latest dev

File: Dockerfile [hunk 1]
```diff
@@ -46,7 +46,7 @@ ENV OPENVPN_USERNAME=**None** \
     WEBPROXY_PORT=8888 \
     WEBPROXY_USERNAME= \
     WEBPROXY_PASSWORD= \
-    DOCKER_LOG=false \
+    LOG_TO_STDOUT=false \
```
Comment (kushagharahi) [inline Dockerfile:49]: https://github.com/haugene/docker-transmission-openvpn/blob/master/Dockerfile#L140

I think this was removed in merge conflicts

**Comment by haugene** [comment]
Looks good :+1: Thanks @kushagharahi

**Pull request merged**